src/rt/sched/status.h

```diff
@@ -119,10 +113,18 @@ namespace verona::rt
       _load_hist |= (_current_load >> 4);
       _current_load = 0;
     }
-  };
 
-  static_assert((sizeof(Status) == 4) && (alignof(Status) == 4));
-  static_assert(
-    (sizeof(Status) == sizeof(std::atomic<Status>)) &&
-    (alignof(Status) == alignof(std::atomic<Status>)));
+    /// Indicates if the token message is in a cown's queue. If zero, a
+    /// new token message should be added if the cown runs another message.
+    inline bool has_token()
+    {
+      return bits::extract<0, 0>(_misc) == 1;
+    }
+
+    /// Set whether the cown has a token in its queue.
+    inline void set_has_token(bool value)
+    {
+      _misc |= (uint8_t)value;
```

Comment:
So
```
set_has_token(false)
```
does nothing?  Shouldn't this unset the bottom bit, but `|` is not your friend.

Reply:
Ah, thank you! I was busy debugging something else that this bug triggered and forgot to push the correct version.